crane-cloud/frontend
# Add admin provision for beta users

# Description
Add provision for admins to select add add beta users

## Type of change


## Trello Ticket ID
https://trello.com/c/fRu1svY1

## How Can This Been Tested?

Pull this branch locally and run it. Check out the admin page for users add select and add a beta user

## Checklist:


# Screenshots
![Screenshot from 2022-05-26 11-57-41](https://user-images.githubusercontent.com/32802973/170455851-170492d1-1df5-431f-aa82-bc08c21b12a2.png)
